canonical-websites/docs.ubuntu.com
# Update ./run to v2.5.0

## Done

Update to latest run script

## QA

Run the usual ./run commands and double check they work:
- `./run clean`
- `./run`
